src/utils/type-transform/SECP256k1-Compression.ts

```diff
@@ -1,6 +1,7 @@
 import secp256k1 from 'tiny-secp256k1'
 import { Convert, combine } from 'pvtsutils'
 import { encodeArrayBuffer, decodeArrayBuffer } from './String-ArrayBuffer'
+import { Buffer } from 'buffer'
```

Comment:
Why not `ArrayBuffer`?

Reply:
The original code is using `Buffer` global without referencing it. It breaks under our new architecture. We may change them later